pyproject.toml

```
@@ -0,0 +1,61 @@
+[build-system]
+requires = ["setuptools>=46.1.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "fhir_migrations"
+version = "0.0.1"
```

we typically try to manage the versioning via git/VCS. using a static version here may conflict with the VCS reports
```suggestion
dynamic = ["version"]
```